src/Snppts/Snippets/SalonMobile.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace Snppts.Snippets
+{
+    public class SalonMobile : IAmASnippet
+    {
+        public string Slug => "salon-mobile";
+        public string Title => "Salon Mobile";
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("kainaalmeida/SalonMobile");
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+
+        public string Description => "This snippet was created to demonstrate some animations";
+
+        public IAmAnAuthor AuthorInfo => new KainaAlmeida();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/kainaalmeida/SalonMobile/blob/master/screenshots/iOS01.png"),
```

Comment:
You will want to use the “raw” urls, so for instance: https://raw.githubusercontent.com/kainaalmeida/SalonMobile/master/screenshots/iOS01.png

Also. Since it supports iOS _and_ Android. Would it be nice to also include the Android screenshot?

Reply:
Of course, I'll adjust